src/main/java/org/alliancegenome/curation_api/controllers/crud/ontology/CHEBITermCrudController.java

```diff
@@ -18,7 +19,10 @@ public class CHEBITermCrudController extends BaseOntologyTermController<CHEBITer
 	@Override
 	@PostConstruct
 	public void init() {
-		setService(chebiTermService, CHEBITerm.class);
+		GenericOntologyLoadConfig config = new GenericOntologyLoadConfig();
+		config.setLoadOnlyIRIPrefix("CHEBI");
+		config.setCalculateClosure(false);
+		setService(chebiTermService, CHEBITerm.class, config);
```

Comment:
@markquintontulloch shouldnt this be in the executor?

Reply:
Around line 202 in the OntologyExecutor?

Reply:
@oblodgett  - err, it is!